src/pages/pages.ts

```diff
@@ -18,7 +18,7 @@ import {Vrv} from "./Vrv/main";
 import {Proxer} from "./Proxer/main";
 import {Novelplanet} from "./Novelplanet/main";
 import {fourAnime} from "./fourAnime/main";
-import {Dreamanimes} from "./Dreamanimes/main";
+import {DreamAnimes} from "./DreamAnimes/main";
```

Comment:
This line throws an error in the build. As the file does not exist. 
Please change the path back to ./Dreamanimes/main or rename the folder to DreamAnimes.

Reply:
Done. Sorry for the inconvenience.